### ISSUE TITLE
Fix date location in mobile view

### ISSUE THREAD

**Issue by whitevo** [Project-Catalyst/project-catalyst.github.io#164]
When I open this https://cardanocataly.st/community-advisor/guide.html page in the android, Honor 10 phone, Brave browser.
I see a date hovering over the actual text.
The same thing actually happens in Win 10, Brave Browser.

![image](https://user-images.githubusercontent.com/10964567/129319125-d2a20de6-475b-4228-9497-6465909f3dc3.png)

Move the updated date to the top right-aligned with the title.

**Comment by tstachl** [comment]
Is this the same issue as #176?

**Comment by whitevo** [comment]
> Is this the same issue as #176?

I think so, not sure where the there update text is located.
Hopefully it someone related to every page rendering, so don't have to fix from all the different places

**Comment by mwojtera** [comment]
Yes this is the same as #176 It occurs in mobile view. It's a plugin in vuepress, afaiu it can be fixed it in one place.
